models/keep3r_network/ethereum/keep3r_network_ethereum_schema.yml

```diff
@@ -0,0 +1,107 @@
+version: 2
+
+models:
+  - name: keep3r_network_ethereum_view_job_migrations
+    meta:
+      blockchain: ethereum
+      project: keep3r_network
+      contributors: [agaperste, wei3erHase]
+    config:
+      tags: ["ethereum", "keep3r_network", "view"]
+    description: >
+      a view logging jobs migrations on Keep3r Network on Ethereum
+    columns:
+      - &event
+        name: event
+        description: "Type of event: LiquidityAddition, LiquidityWithdrawal, CreditsReward, KeeperWork, JobMigrationIn, JobMigrationOut"
+      - &evt_index
+        name: evt_index
+        description: "Index of the event"
```

Comment:
I think this is a redundant description. I think it's better to remove it. We should just add descrptions when we think it will meaningfully improve the end users understanding of the table.